tdrs-backend/tdpservice/users/views.py

```diff
@@ -43,7 +43,7 @@ def get_serializer_class(self):
             "set_profile": SetUserProfileSerializer,
         }.get(self.action, UserSerializer)
 
-    @action(methods=["POST"], detail=False)
+    @action(methods=["PATCH"], detail=False)
```

Comment:
`PATCH` is the appropriate RESTful action for updating a record partially. The user experience is that this is an account creation, but since that action is taken at login, at the system level, it's an update.